aicon/backend/server.py

```diff
@@ -73,8 +73,8 @@ def run(self, client_data: Dict[str, Union[str, Queue]]):
         data: Dict[str, str] = {
             "client_uuid": self.client_uuid,
             JSON_CURRENT_ITER: str(total_iteration),
-            JSON_IMG_PATH: f"/home/aicon/img/{self.client_uuid}/{i}",
-            JSON_GIF_PATH: f"/home/aicon/gif/{self.client_uuid}/{i}",
+            JSON_IMG_PATH: f"../static/dst_img/{client_data[JSON_MODEL_NAME]}/{self.client_uuid}/{total_iteration:06d}.png",
```

Comment:
Be careful.
![CatYouGotThisGIF](https://user-images.githubusercontent.com/80256066/132814692-cb58763d-2945-4acc-a28f-68b57775e214.gif)